README.md

```diff
@@ -2,19 +2,21 @@
 <img src="https://i.postimg.cc/xTpWgq3L/pixiu-logo.png"  width="100%" height="100%">
 </p>
 
+- [Spainish](README.es.md)
```

Comment:
```suggestion
```

Reply:
Maybe we can move it to checkpoints, write like:

**Languages**
- [English](README.md)
- [Spanish](README.es.md)